contracts/pools/SyndicateCorePool.sol

```diff
@@ -90,6 +94,15 @@ contract SyndicateCorePool is SyndicatePoolBase {
         return weightToReward(user.totalWeight, vaultRewardsPerWeight) - user.subVaultRewards;
     }
 
+    /**
+     */
+    function setQuickReward(uint _quickReward) external {
+        // verify function is executed by the factory owner
+        require(factory.owner() == msg.sender, "access denied");
+        require(_quickReward < 100000, "parameter out of range");
```

Comment:
It looks like this a function called by us from outside.  The check is to be sure that at most it is the 10%, right?

Reply:
if staked for a year, the check makes sure it's not more than 10x of the staked amount in sSYN.  We will probably use a much small number in reality.